Web_Development/FrontEnd/HTML_IFrames/iframes.md

```diff
@@ -0,0 +1,97 @@
+# HTML IFrames (Inline Frames) 
+An IFrame (Inline Frame) is an HTML document embedded inside another HTML document on a website.<br>
+The IFrame HTML element is often used to insert content from another source,such as an advertisement, into a Web page.<br>
```

Comment:
Do not use HTML tags, just keep it with Markdown only. Use line spaces around the line to make it as paragraphs.